Source/WebKit/WebProcess/GPU/media/AudioTrackPrivateRemote.cpp

```diff
@@ -60,22 +61,29 @@ void AudioTrackPrivateRemote::updateConfiguration(AudioTrackPrivateRemoteConfigu
 {
     if (configuration.trackId != m_id) {
         m_id = configuration.trackId;
-        if (client())
-            client()->idChanged(m_id);
+        notifyClients([id = m_id](auto& client) {
```

Comment:
We don't need to crossThreadCopy() m_id here? I didn't check the type. I just thought it was worth asking because you're doing it for m_id later in this PR.

Reply:
it's an int